lib/src/arithmetic/values/fixed_point_value.dart

```diff
@@ -143,18 +143,58 @@ class FixedPointValue implements Comparable<FixedPointValue> {
     return compareTo(other) == 0;
   }
 
-  /// Constructs [FixedPointValue] of a Dart [double] rounding away from zero.
+  /// Return a string representation of FloatingPointValue.
+  ///  return sign, exponent, mantissa as binary strings.
+  @override
+  String toString() =>
+      "(${signed ? '${value[-1].toString(includeWidth: false)} ' : ''}"
+      "${(m > 0) ? '${value.slice(m + n - 1, n).bitString} ' : ''}"
```

Comment:
why does the `m` piece not use `.toString(includeWidth: false)`?

Reply:
bitString = toString(includeWidth:false).    In this case, I run into the 80 character margin with the interpolated string.